.github/workflows/check-examples.yml

```diff
@@ -1,9 +1,9 @@
 name: check examples
 
 on:
-  schedule:
-    # 04:00 every weekday morning
-    - cron: '0 4 * * 1-5'
+  pull_request:
+    branches:
+      - master
```

Comment:
Why only for master and not for develop?

Reply:
I think it would be good to keep the weekly execution to get notified about problems that occurs due to changes in other modules.